### ISSUE TITLE
Combine with another Python script

### ISSUE THREAD

**Issue by fthmhmt** [jnweiger/led-badge-44x11#11]
Thank you for sharing the code. It is working great!
I am not familiar with python and would like to ask you if you can help me out to combine your code with another one.

An import into your code of this code (https://github.com/Zacki84/Wien-Abfahrtsmonitor) did not give the output to the LED Badge.

Can you give me here a tip how to print output of codes to the LED Badge?

**Comment by jnweiger** [comment]
The basic call sequence in the current version is something like this:
```
(data, cols) = bitmap_text("Hello World!")
head = header([cols], 4, 0, 0, 0)
pyhidapi.hid_write(dev, head+data)
```

You need to cut/copy/paste so you have all the parts that are needed, and probably also have to remove some unneeded parts (e.g. the option parser).
All this would be much easier, if the code were a proper python module, of course. Blame me for being lazy in the first place ....

**Comment by semaf** [comment]
> All this would be much easier, if the code were a proper python module, of course. Blame me for being lazy in the first place ....

You are right. As module it would be really easier. Hope that you have time to do it very soon.

**Comment by jnweiger** [comment]
Well, I don't intend to save the world all by myself... Patches welcome.

**Comment by bensartori** [comment]
Hello fthmhmt, there were progress on this topic..., see last comment in #3 and PR #45 .

**Comment by bensartori** [comment]
Hello @fthmhmt, does it work for you, now (see last comment)?
